src/AWSS3.jl

```diff
@@ -677,7 +690,9 @@ function s3_delete_tags(aws::AbstractAWSConfig, bucket, path=""; kwargs...)
     return parse(r)
 end
 
-s3_delete_tags(a...; b...) = s3_delete_tags(global_aws_config(), a...; b...)
+function s3_delete_tags(bucket, path=""; kwargs...)
+    return s3_delete_tags(global_aws_config(), bucket, path; kwargs...)
+end
```

Comment:
Same issue as above:
```suggestion
s3_delete_tags(args...; kwargs...) = s3_delete_tags(global_aws_config(), args...; kwargs...)
```